WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -17,6 +17,19 @@ class RemoteFeatureFlagStore {
         return queue
       }()
 
+    /// The `deviceID` ensures we retain a stable set of Feature Flags between updates. If there are staged rollouts or other dynamic changes
+    /// happening server-side we don't want out flags to change on each fetch, so we provide an anonymous ID to manage this.
```

Comment:
typo: "out flags" -> "our flags" I suppose?
But I think you've only moved this code.

Reply:
> But I think you've only moved this code.

Correct, but I'll fix the typo.